### ISSUE TITLE
Resizing Headers for Minimal Theme

### ISSUE THREAD

**Issue by SamAdams51** [kepano/obsidian-minimal-settings#12]
Hi kepano,
I enjoy the Minimal Theme for quite a while and there is only the header sizes that I find to be to small. In order to adjust this, I use the „Style Settings“ plug-in. However, it leads to different header sizes in edit and preview mode. Therefore it is just a workaround for me but not a good one. Unfortunately I don’t know how to write a css-snippet to change the header sizes. Could you consider to put the ability to change the header sizes into the „Minimal Theme Settings“ plug-in?

Thanks for considering

**Comment by kepano** [comment]
Are you saying you want the headings to be the same size in Edit and Preview, or different sizes?

Changing those via Style Settings should give you the same size. If you want different sizes the short term solution would be to use a snippet

**Comment by SamAdams51** [comment]
When I work in Obsidian, I spend most of my time in edit mode. Therefore, I want the appearance of the notes in Edit mode to be as close as possible to the appearance in Preview mode.
This means that the headings in Edit mode should have the same size and formatting as they have in Preview mode. The attachment shows my settings for the Typography in the Minimal Theme Settings plug-in.

So from your mails, I understand, that they should look the same in Edit and Preview and they do except for heading 3.

**Comment by kepano** [comment]
Hi Sam, I am not able to see attachments. You'll need to go into the issue on Github and upload them here. I don't think you can do this via email.

Minimal is designed to do what you are saying by default. The sizes are the same. I suspect that you have some kind of conflict, e.g. running both a snippet and Style Settings causing the difference in font sizes.

**Comment by SamAdams51** [comment]
Hi Stephan,
sorry, but I didn’t know that you can’t see the attachments in my E-Mails. I attach them to this comment. Meanwhile, I experimented a little with different fonts (selected in the Minimal Theme Settings plug-in). When I did the screen shots of the sample note I had the iA Writer Duo font selected and the difference between Eedit and Preview is quite obvious. Currently I use the Apple System font for Preview and Edit and here the difference is not as big as with the iA Writer font.

Thanks again for taking your time to help.
Best regards.
![Bildschirmfoto 2021-11-11 um 16 13 26](https://user-images.githubusercontent.com/42751459/141353478-2a4750b1-c9f9-4d82-a132-26f516488531.png)
![Bildschirmfoto 2021-11-11 um 16 13 53](https://user-images.githubusercontent.com/42751459/141353482-490e442b-6d22-4195-b51d-216c36cf3823.png)
![Bildschirmfoto 2021-11-11 um 19 11 26](https://user-images.githubusercontent.com/42751459/141353483-738e9966-7c2e-4786-82d4-4c8ac44b9d38.png)

**Comment by kepano** [comment]
There are a few issues I am noticing here

1. I think that you do not have the iA Writer fonts installed on your system. You will need to download those and install them if you want to use them.
2. If you are using a different font in Edit and Preview mode, fonts may appear to be different sizes visually, even if the actual numeric size is the same. This is because they may have a different x-height and other properties of the font give that effect.
3. Please note that simultaneously running Minimal Theme Settings and Style Settings and snippets can cause conflicts which become difficult to debug because they can all be used to modify the same values

**Comment by SamAdams51** [comment]
Hi Stephan,

- to 1. You are right, at the time I took the screen shots, iA Writer font was not installed. But I also experimented with fonts which were installed on my Mac and they showed the same behavior, some more some less.
- to 2. I always used the same font for Edit and Preview.
- to 3. At the time I took the screen shots, I had the Style Settings App already uninstalled. So I had the snipped you see in the screen shots and Minimal Theme Settings active at the same time. Until you probably add a feature 😀 in the Minimal Theme Settings, the snippet is my only way to modify the header sizes. So for the time being I have deactivated the Minimal Theme Settings plug-in and just use the snipped.

Maybe you find some time in the future to add that feature to the Minimal Theme Settings plug-in. It would be super cool because I love your Theme.

Best regards, happy coding and thanks again for your time.

Helmut

**Comment by kepano** [comment]
Can you update to the latest version of Minimal? I may have fixed the issue for you.

**Comment by SamAdams51** [comment]
I updated the theme but don’t see a difference. Do you want me to deactivate the snippet and reactivate the plug-in?

**Comment by SamAdams51** [comment]
Hi Stephan,
Its me again and I have important news. When we communicated yesterday evening (German time), I was on my iPad. This morning I went back to my MacMini and found, that things are different between iPad and MacMini.

On my MacMini, I updated the Minimal Theme also and I have the Minimal Theme Settings plug-in and the Header-Sizes.css snippet activated. Everything is super now on this device, Edit and Preview are beautiful and look the same.😀👍.
On the iPad however, with the same setup, Preview looks good but in Edit the headings are way to big. So my solution for now is to have a different config file (.obsidian.iPad) for the iPad and I disabled the Header-Sizes.css snippet for the iPad. The result is not as super good as on the MacMini (the headers are all a little bit too small but at least have the same size in Edit and Preview) but I can very well live with it.

So, I apologize if I confused things and thank you very much again for all the effort you put into this issue.

Best regards and have a nice weekend

Helmut
